app/assets/javascripts/discourse/app/components/d-editor.js

```diff
@@ -223,6 +222,14 @@ export function onToolbarCreate(func) {
   addToolbarCallback(func);
 }
 
+const COMPOSER_EVENTS = [
```

Comment:
I realise it's tricky now, but it would be way easier to review (and `git blame` later) if any unrelated refactoring was split into a separate PR. A common technique I use is to stack the PRs: make a refactoring branch, and then base your feature branch on that.

Then we can review/merge the no-op refactor first, rebase the feature PR on `main`, and then review/merge the feature.

Reply:
Removed all unrelated refactorings.